Taskfile.yml

```diff
@@ -62,14 +98,26 @@ tasks:
           INPUT_LOG_LEVEL: '{{.CLI_ARGS}}'
 
   build:
+    desc: Build the docker image; set PLATFORM env var for cross-platform builds
+    vars:
+      PYTHON_PLATFORM: '{{.PYTHON_PLATFORM}}'
     cmds:
       - task: bash:build
+        vars:
+          PLATFORM: '{{if eq .PLATFORM "all"}}{{.SUPPORTED_PLATFORMS}}{{else if .PLATFORM}}{{.PLATFORM}}{{else}}{{.LOCAL_PLATFORM}}{{end}}'
+          DOCKER_BUILD_CUSTOM_ARGS: '--build-arg PYTHON_PLATFORM="{{.PYTHON_PLATFORM}}"'
```

Comment:
This is a new feature I needed for this situation but will also be helpful in the future.  It's susceptible to injections, but projects can only inject to their own pipelines, when they have edit access to the workflow.yml/taskfile.yml so it shouldn't matter